CONTRIBUTING.org

```diff
@@ -20,11 +20,23 @@ make plain
 #+end_src
 
 * Contributing code
-The basic code style guide is to use:
+Here are the important settings:
 #+begin_src elisp
 (setq indent-tabs-mode nil)
+(setq lisp-indent-function #'common-lisp-indent-function)
 #+end_src
-It is provided for you in [[https://github.com/abo-abo/swiper/blob/master/.dir-locals.el][.dir-locals.el]], please obey it.
+
+The setting for =indent-tabs-mode= is automatically applied by
```

Comment:
Shouldn't code be marked up with tildes? I.e. `~indent-tabs-mode~`.

Reply:
The rest of the file uses `=` monospace rather than `~` code. I could change that, too, though.

Reply:
> The rest of the file uses `=` monospace rather than `~` code. I could change that, too, though.

![fix-all-things](https://user-images.githubusercontent.com/9121222/40623275-f1941d72-629d-11e8-938a-8c79dc900de1.png)

Silliness aside, I'd like to know whether there's a good reason to use `=` for everything. If there isn't, then I'd prefer if `~` were used where appropriate. This can, of course, be done at a later time, but if we agree on using `~`, then we might as well use it for any new additions.

Reply:
Actually, within many of my projects I've made an ad-hoc convention that `~...~` should be used as equivalent of `<kbd>...</kbd>` in Markdown. With a bit of customization, these tags will be properly exported both in HTML and in Info. So I'd like to keep `~...~` exclusively for keyboard shortcuts.

Reply:
Fair enough.